src/core/or/circuitpadding.c

```diff
@@ -1165,6 +1159,12 @@ circpad_machine_schedule_padding,(circpad_machine_runtime_t *mi))
   struct timeval timeout;
   tor_assert(mi);
 
+  /* Don't schedule padding if we are currently in dormant mode. */
+  if (net_is_completely_disabled()) {
```

Comment:
This isn't a test for dormant mode either; instead look at is_participating_on_network()?